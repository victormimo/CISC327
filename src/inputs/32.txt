login
machine
B
9023923
50000
B
9023923
50000
B
9023923
50000
end