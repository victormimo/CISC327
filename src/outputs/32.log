Welcome to QBASIC.
Please type login.
Please login as: machine or agent
Please select a transaction or type logout: 
A. deposit to an account
B. withdraw from an account
C. transfer from one account to another
Please enter the letter or type logout: 
Please enter account number: 
Please enter the amount to withdraw in cents: 
Please select a transaction or type logout: 
A. deposit to an account
B. withdraw from an account
C. transfer from one account to another
Please enter the letter or type logout: 
Please enter account number: 
Please enter the amount to withdraw in cents: 
Please select a transaction or type logout: 
A. deposit to an account
B. withdraw from an account
C. transfer from one account to another
Please enter the letter or type logout: 
Please enter account number: 
Please enter the amount to withdraw in cents: 
a total of at most $1,000 can be withdrawn from a single account in a single ATM session
Please select a transaction or type logout: 
A. deposit to an account
B. withdraw from an account
C. transfer from one account to another
Please enter the letter or type logout: 
