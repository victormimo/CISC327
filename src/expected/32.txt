WDR 9023923 50000.0 (none) Beer Store
WDR 9023923 50000.0 (none) Beer Store
EOS
